package.json

```diff
@@ -7,7 +7,7 @@
     "url": "https://github.com/onmodulus/rabbit-pubsub/issues"
   },
   "devDependencies": {
-    "@modulus/standard": "0.6.0",
+    "standard": "7.1.2",
```

Comment:
I don't understand. It's always been `@modulus/standard` and should continue to be, yeah...?